### DISCUSSION TITLE
2 channel vs 16 channel

### DISCUSSION THREAD

**Discussion by BruceAC** [ExistentialAudio/BlackHole#290] [Q&A]
As a new user of Blackhole, I am curious regarding all the differences between 2 and 16 channel other than the number of channels.  Is 16 channel intended for MIDI and 2 channel for analog?   I am not connecting any external audio devices to my Mac, so is 2 channel the best choice or is there some possible reason I should install the 16 channel version?  If I want to record 2channel streaming or system audio and simultaneously record from the built in microphone, do I need more than 2 channels or do I just enable a multi input or output device to record both inputs?

**Comment by devinroth** [comment]
No difference except the number of channels.

But some apps handle 2ch different than 16ch.

**Reply by threesixtycreative** [reply]
Hi Devin

I think the crux of the question for newbies is, which version should a general user install?

**Comment by devinroth** [comment]
2ch.

Maybe I should make a note on the website.

Devin

**Reply by threesixtycreative** [reply]
Thanks for the quick reply, Devin!

**Comment by yuriteixeira** [comment]
Hey @devinroth 👋 Would you mind to leave in the docs some example use-cases where 16ch would be useful?

**Comment by devinroth** [comment]
Basically any time you are using more than 2 channels, 5.1 or 7.1 or sending audio between daws. For most people 2ch is all they need. Stick with 2 unless you know that you need more.

Devin Roth

**Comment by joakimechio** [comment]
Hi Devin,
Is there a way i can send different audio sources, for instance an input from my soundcard and the output from my DAW to Blackhole 16ch and get those separately, like you can do with Loopback for instance? I need to have a channel with mic (plugged to my soundcard) and one with my DAW in OBS and I can't figure out how to do it with Blackhole.

**Comment by drewbk99** [comment]
@devinroth Would using the 16ch version create much larger screen recording file sizes in Quicktime Player than using the 2ch version, even if only 2 channels of audio would be played in both instances? Not sure if the 16ch version would write a bunch of empty data for the 14 unused channels in the recording file or anything like that.

**Comment by devinroth** [comment]
16 channels of uncompressed audio but always be larger than two channels. Whether or not you actually record those 14 unused channels depends on the app that is recording. I can’t speak to that specifically.

Devin
